CMakeLists.txt

```diff
@@ -287,6 +287,10 @@ The %{name} package also contains HTML formatted documents containing
 hardening guidances that have been generated from XCCDF benchmarks.
 ")
 
+include(CTest)
+FILE(GLOB_RECURSE all_datastreams "ssg-*-ds.xml")
+add_test(STIG_IDS_PRESENCE python "${CMAKE_CURRENT_SOURCE_DIR}/shared/utils/stig-scanner.py" ${all_datastreams})
```

Comment:
the test name looks like a variable because you used all caps
you didn't use all caps for the actual variable all_datastreams and you didn't prefix it

Reply:
I have changed the variable case, but I don't know what is meant by prefixing a variable.